core/domain/exp_domain_test.py

```diff
@@ -524,7 +524,8 @@ def test_validation(self):
             'tagged_skill_misconception_id': None
         })
 
-        init_state.update_interaction_answer_groups(old_answer_groups)
+        init_state.update_interaction_answer_groups(
+            state_domain.AnswerGroup.from_dict(old_answer_groups))
```

Comment:
`state_domain.AnswerGroup.from_dict` accepts dict as a parameter and `old_answer_groups` is a list, right? Have you test this locally?

Reply:
@DubeySandeep I have changed them to dict

Reply:
@DubeySandeep Can you help me how can I pass two or more dictionaries to from_dict, I am facing a problem there, can you help me sort it out. I got struck

Reply:
maybe we can do:
```
answer_groups = [state_domain.AnswerGroup.from_dict(answer_group_dict) in answer_groups_list]
init_state.update_interaction_answer_groups(answer_groups)

```